Added Intros for REST API Pages (#10001)
github/docs: content/rest/reference/billing.md

---
title: Billing

> intro: 'With Billing API, you can monitor the {% data variables.product.prodname_actions %} and {% data variables.product.prodname_registry %} charges for individual and your organization.'

Reviewer: ```suggestion
intro: 'With the Billing API, you can monitor the charges and usage {% data variables.product.prodname_actions %} and {% data variables.product.prodname_registry %} for a user or organization.'
```

> intro: 'With Billing API, you can monitor the {% data variables.product.prodname_actions %} and {% data variables.product.prodname_registry %} charges for individual and your organization.'

Reviewer: Again, let's remove the nearly identical sentence in the body of this article.
